ci-operator/step-registry/upi/deprovision/OWNERS

```diff
@@ -0,0 +1,11 @@
+approvers:
+- smarterclayton
+- wking
```

Comment:
I don't think I'm specifically qualified to be a deprovision approver, and you already have me in the parent `upi/OWNERS`.  Can you drop me here?